source/features/rgh-improve-new-issue-form.tsx

```diff
@@ -0,0 +1,36 @@
+import React from 'dom-chef';
+import select from 'select-dom';
+import * as pageDetect from 'github-url-detection';
+
+import features from '.';
+import clearCacheHandler from '../helpers/clear-cache-handler';
+import {isRefinedGitHubRepo} from '../github-helpers';
+
+function init(): false | void {
+	if (new URL(location.href).searchParams.get('template') !== '1_bug_report.yml') {
+		return false;
+	}
+
+	const {version} = browser.runtime.getManifest();
+	select('input#issue_form_extension_version')!.value = version;
+	select('input[id*="extension_cache"]')!.parentElement!.after(
+		<button
+			className="btn"
+			type="button"
+			onClick={clearCacheHandler as unknown as React.MouseEventHandler}
+		>
+			Clear cache
+		</button>,
+	);
+}
+
+void features.add(import.meta.url, {
+	asLongAs: [
+		isRefinedGitHubRepo,
+	],
+	include: [
+		pageDetect.isNewIssue,
+	],
```

Comment:
I think these two should be replaced by a single check for the specific template instead of adding it inside `init`